app/main/views/briefs.py

```diff
@@ -194,6 +195,90 @@ def update_brief_status(brief_id):
     return jsonify(briefs=brief.serialize()), 200
 
 
+@main.route('/briefs/<int:brief_id>/<any(publish, withdraw):action>', methods=['PUT'])
+def update_brief_status_by_action(brief_id, action):
+    updater_json = validate_and_return_updater_request()
+
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).first_or_404()
+
+    if brief.framework.status != 'live':
+        abort(400, "Framework is not live")
+
+    action_to_status = {
+        'publish': 'live',
+        'withdraw': 'withdrawn'
+    }
+    if brief.status != action_to_status[action]:
+        brief.status = action_to_status[action]
+
+        validate_brief_data(brief, enforce_required=True)
```

Comment:
I think we only require this when going from draft to published. I'm not necessarily against rerunning the validation when withdrawing a brief, but it would commit us to _always_ running migrations on published briefs when we change the schemas.